src/Datasource/FixtureInterface.php

```diff
@@ -16,7 +16,7 @@
  */
 namespace Cake\Datasource;
 
-use Cake\Database\StatementInterface;
+use Cake\Database\Statement;
```

Comment:
This is a separate issue but datasource package shouldn't have a dependency on a database package class :disappointed: